src/odemis/gui/model/__init__.py

```diff
@@ -31,6 +31,7 @@
 from odemis.acq import path, leech
 import odemis.acq.stream as acqstream
 from odemis.acq.stream import Stream, StreamTree, StaticStream, RGBSpatialProjection, DataProjection
+from odemis.dataio import catmaid
```

Comment:
It'd be really nice that this import is not needed. I'd rather not have the GUI depend on dataio.catmaid.